AppCenterAnalytics/AppCenterAnalytics.xcodeproj/project.pbxproj

```diff
@@ -1660,6 +1660,7 @@
 			baseConfigurationReference = 049BC82A1ECE3B9400FB6719 /* iOS.xcconfig */;
 			buildSettings = {
 				DEVELOPMENT_TEAM = "";
+				SUPPORTS_MACCATALYST = YES;
```

Comment:
Changes in project files are not allowed, just change (it's already there) this value in xcconfig file